relational/table.py

```diff
@@ -52,6 +54,48 @@ def nv_row(self, proj, name='', value='', data=''):
         return row
 
 
+def _get_mohcd_units(proj, source_override=False):
+    """
+    Gets net new units and bmr counts from the mohcd dataset.  Prioritizes
+    data from MOHCDPipeline, and falls back to MOHCDInclusionary if none
+    found.
+
+    If source_override is specified, will pull numbers only from the given
+    source.  Otherwise will pull from pipeline and inclusionary mohcd data.
+    Nothing technically prevents you from providing a non-mohcd source, but
+    you will probably not get useful numbers.
+
+    Returns:
+      A tuple of (number units, number of BMR units, source) from MOHCD, or
+      None if nothing found.
+    """
+    sources = [source_override] if source_override else [
+        MOHCDPipeline.NAME,
```

Comment:
Nit: we log the name from the source here eventually so instead of logging OUTPUT_NAME we'd just log NAME?

Reply:
yer right. i didn't think of a better OUTPUT_NAME to use, so it's set to NAME in the sources file, but at least now it's consistent